common/artifacts/000_wc_artifacts.txt

```diff
@@ -568,6 +572,27 @@ runeblade_unholy_1 = {
 	slot = weapon
 }
 
+### One-handed weapon ###
+
+# Rare
+meteor_shard = {
+	quality = 3		# 3
+	flags = { combat_weapon dagger }
+	
+	active = {
+		is_adult = yes
+	}
+
+	stacking = no
+
+	martial = 1
+	combat_rating = 10
+
+	picture = "GFX_inv_weapon_shortblade_29"
```

Comment:
I have no idea why you added `inv_weapon_shortblade_25` but used `inv_weapon_shortblade_29` here

Reply:
Good catch, the template was copy-pasted and I missed that line.